### PR TITLE
Update overview-daniel.md

### PR DESCRIPTION
**Pull request by deullmer**
Create overview-daniel.md

### CODE REVIEW

File: docs/overview-daniel.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+---
+
+# To-Do service API overview
+
+The To-Do service is a simple yet powerful cloud-based REST API interface for helping busy people stay on track with their obligations and manage their schedules. If, for example, your days are filled with meetings and parenting duties, the To-Do service can ease your mind from worries about missing anything important.
```
Comment (rbwatson) [inline docs/overview-daniel.md:7]:
```suggestion
The To-Do service is a simple, yet powerful, cloud-based REST API interface for helping busy people stay on track with their obligations and manage their schedules. If, for example, your days are filled with meetings and parenting duties, the To-Do service can ease your mind from worries about missing anything important.
```
Just some copy edits

File: docs/overview-daniel.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+With the To-Do service, adding a user profile takes only a minute. You can then easily add tasks to their profile. Task properties can include automatic reminders, detailed task descriptions, and warning level information. Users can access the list of tasks assigned to them at any time.
```
Comment (rbwatson) [inline docs/overview-daniel.md:9]:
```suggestion
With the To-Do service, adding a user profile takes only a minute.
You can then easily add tasks to their profile.
Task properties can include automatic reminders and detailed task descriptions.
Users can access the list of tasks assigned to them at any time.
```
The `warning` field in the `task` resource specifies how long before the task to send a reminder. Apparently, "warning" isn't the most obvious name for what the property does (other's have made a similar assumption). I'll talk to the designer :) 

Also, starting sentences on a new line as in the suggestion makes it easier to review edits.

File: docs/overview-daniel.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+## Get started
+
+Learn how to add and manage user and task resources. This section provides tutorials to help developers quickly start working with the To-Do service API.
+
+**Note:** Before you can start the tutorials, you need to make sure that you have the prerequisite tools, and then you need to set up your development system to work with the To-Do service. The following topic covers this information:
+
+* [Before you start a tutorial](tutorials/before-you-start-a-tutorial.md)
+
+After your system is ready, these tutorials show you how to perform the most common tasks:
+
```
Comment (rbwatson) [inline docs/overview-daniel.md:22]: The following list would benefit from some organization.
For example by  first tasks, then additional features, or by resource, or by use-case, etc.

**Review by rbwatson — CHANGES_REQUESTED** [review]
This is great.

Don't forget to link to it from `index.md`.

File: docs/overview-daniel.md (hunk 3 above)
Comment (deullmer) [reply docs/overview-daniel.md:22]: Done.

**Review by rbwatson — APPROVED** [review]
rebased and squashed for merge
link testing found broken link
fixed broken link
LGTM

**Pull request merged**